Makefile

```diff
@@ -20,7 +20,7 @@
 CPUS      := $(shell getconf _NPROCESSORS_ONLN 2>/dev/null || getconf NPROCESSORS_ONLN 2>/dev/null || echo 1)
 
 PREFIX    = "/usr/local"
-VERSION   := $(cmake -P config.cmake | sed -n -e "s/^.*VERSION  *//p")
+VERSION   := $(shell cmake -P config.cmake | sed -n -e "s/^.*VERSION  *//p")
```

Comment:
This change is in 3.5rc and in 3.x, so not an issue, but why then is it listed here?